concepts/Code_status/carePlan_getItemValues.Rmd

```diff
@@ -0,0 +1,34 @@
+---
+title: "carePlan_getItemValues"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Connect to database.
+
+```{r}
+require("RPostgreSQL")
+drv <- dbDriver("PostgreSQL")
+con <- dbConnect(drv, dbname = "eicu",
+                 host = "localhost", port = 5432,
```

Comment:
If possible avoid including your own username etc here. Might be better to include config settings at the top to make it easier for people to reuse the code. In [other examples](https://github.com/MIT-LCP/mimic-code/blob/master/notebooks/rmd_example/mimic_los.Rmd), we've included this at the top:

```
# Load configuration settings
dbdriver <- 'PostgreSQL'
host  <- '127.0.0.1'
port  <- '5432'
user  <- 'postgres'
password <- 'postgres'
dbname <- 'mimic'
schema <- 'mimiciii'
```

Reply:
good idea.